src/Unzer.php

```diff
@@ -1059,5 +1061,10 @@ public function debugLog($message): void
         }
     }
 
+    public function hasProductionKey(): bool
```

Comment:
I think its not a problem to expose it since it does not reveal anything and its used by other services. However it might be better to move the logic out of the unzer facade to have it cleaner. I will update that.

Reply:
Change is done, please have a look again.